hips/utils/wcs.py

```diff
@@ -46,20 +52,20 @@ class WCSGeometry:
     CDELT : -1.0  1.0
     NAXIS : 0  0
     >>> wcs_geometry.shape
-    Shape(ny=10, nx=20)
+    Shape(height=10, width=20)
     """
     WCS_ORIGIN_DEFAULT = 0
     """Default WCS transform origin, to be used in all WCS pix <-> world calls."""
 
-    def __init__(self, wcs: WCS, shape: tuple) -> None:
+    def __init__(self, wcs: WCS, height: int, width: int) -> None:
```

Comment:
I'd suggest we always use the order: width, then height , i.e. please change this here.